src/Http/Http/src/StreamResponseBodyFeature.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.Http
+{
+    /// <summary>
+    /// An implementation of <see cref="IHttpResponseBodyFeature"/> that aproximates all of the APIs over the given Stream.
+    /// </summary>
+    public class StreamResponseBodyFeature : IHttpResponseBodyFeature
```

Comment:
StreamResponseBodyFeature  => DefaultResponseBodyFeature ?

Reply:
Already answered elsewhere.

Reply:
Nvrm, apparently I gave this feedback before